tests/test_content.py

```diff
@@ -421,6 +421,83 @@ def test_datetime():
     assert new_result == '2001-02-03 04:05'  # output
 
 
+def test_nested_align():
+    """
+    # Nested Formats
+
+    Additionally, new style formatting allows all of the components of
+    the format to be specified dynamically using nested formats.
+
+    Nested formats can be applied to every portion of the format after
+    the colon. Variable names can not be set dynamically because double
+    braces are the escape characters for braces, so something like
+    `'{{which}}'.format(0.1, 0.2, which=1)` would result in `'{which}'`,
+    not in `'0.2'`. Braces are not allowed in attribute names and item
+    indices are passed literally, so nested formats can not be used
+    anywhere before the colon in a format.
+
```

Comment:
I feel this paragraph doesn't really help the explanation. Maybe just the first sentence would be enough.